lib/keel/gcloud/kubernetes/replication_controller.rb

```diff
@@ -74,6 +73,16 @@ def self.replace file
         Cli.new.system_call "kubectl replace -f #{file}"
       end
 
+      # 
+      # Create a Deployment and expose it on kubernetes
+      #
+      def self.create namespace, app_name, project_id, port, sha
+        command   = "kubectl run #{app_name} --image=gcr.io/#{project_id}/#{app_name}:#{sha} --namespace=#{namespace}"
```

Comment:
gcr.io is gcloud specific, which we are basically tied to (though supporting generic kubernetes would be good as well, at some point).